models/branches.go

```diff
@@ -173,6 +173,20 @@ func (protectBranch *ProtectedBranch) GetGrantedApprovalsCount(pr *PullRequest)
 	return approvals
 }
 
+// GetRejectedReviewsCount returns the number of rejected reviews for pr.
+func (protectBranch *ProtectedBranch) GetRejectedReviewsCount(pr *PullRequest) int64 {
```

Comment:
The method could be a method or `PullRequest`.

Reply:
@oscarcosta protectBranch is never used so it should not be a function of it ..